backend/editor/github_functions.py

```diff
@@ -135,3 +135,17 @@ async def create_pr(self, description) -> PullRequest:
                 *self.repo_info, title=title, body=body, head=self.branch_name, base="main"
             )
         ).parsed_data
+
+    async def is_pr_merged(self, pr_number: int) -> bool:
+        """
+        Check if a pull request is merged
+        """
+        try:
+            await self.connection.rest.pulls.async_check_if_merged(
+                *self.repo_info, pull_number=pr_number
+            )
+            return True
+        except RequestFailed as e:
+            if e.response.status_code == 404:
+                return False
```

Comment:
Why False in this case ? Do you think it might means PR is not yet created ? Maybe a comment can help here.

Reply:
![Screenshot 2024-02-06 at 19 54 28](https://github.com/openfoodfacts/taxonomy-editor/assets/77902715/d04401ae-b306-4c01-a3bc-2a9ecc429b24)
I've bumped the `githubkit` dependency so that there is a link to the `github` docs, that details the status code return values